unittests/rem_voting_test.cpp

```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
          // rem weight:      0.678;
          // staked:          799999999000
          const auto prod = get_producer_info( name("proda") );
-         BOOST_TEST_REQUIRE( 8.8559892136843136e+17 == prod["total_votes"].as_double() );
+         BOOST_TEST_REQUIRE( 8.4652838071982426e+17 == prod["total_votes"].as_double() );
       }
 
       // 8 Weeks Later
       {
          produce_min_num_of_blocks_to_spend_time_wo_inactive_prod(fc::days( 7 * 8 )); // +8 weeks
          votepro( N(whale1), { N(proda) } );
 
-         // weeks to mature: 0;
+         // weeks to mature: 0;1.3925604968379901e+18
          // eos weight:      1.811133596417372e+06;
          // rem weight:      1.000000;
          // staked:          799999999000
          const auto prod = get_producer_info( name("proda") );
-         BOOST_TEST_REQUIRE( 1.4489068753227643e+18 == prod["total_votes"].as_double() );
+         BOOST_TEST_REQUIRE( 1.3925604968379901e+18 == prod["total_votes"].as_double() );
 
       }
```

Comment:
add one more test where "weeks to mature" = 0

Reply:
Done.